msg/NodeState.msg

```diff
@@ -1,7 +1,7 @@
 # Array of nodes to be traversed for fulfilling the order
 
 string nodeId                                       # Unique node identification 
-int32 sequenceID                                    # sequenceId to discern multiple nodes with same nodeId.
+int32 sequenceId                                    # sequenceId to discern multiple nodes with same nodeId.
```

Comment:
wasn't that fixed already?

Reply:
hmm, weird

Reply:
ok, my master was outdated and had that fix manually, force-pushed to correct